### PR TITLE
Milestone 2: Decorate class

### PR DESCRIPTION
**Pull request by yetemegn-telaye**
### School library system milestone 2 decorate class

**In this milestone I:**

- [ ] Implemented composition 
- [ ] Created base decorator
- [ ] Created capitalize decorator
- [ ] Created trimmer decorator

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Yetemegn, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented, and describe between 1-3 **key features** of the application.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -101,9 +101,10 @@ FUTURE FEATURES
 
 ## 🔭 Future Features <a name="future-features"></a>
 
->
 
-- [ ] **[I would like to add more data in the future]**
+- [ ] Checks if a person has correct name
+- [ ] Capitalizes name
+- [ ] Trims a persons name
```
Comment (rotimiazeez) [inline README.md]: - Kindly add this info to your `Key Feature` section(line 63) and add 1 - 3 features you'll add in the future to this section 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @yetemegn-telaye 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by DeliceLydia — APPROVED** [review]
## Hi @yetemegn-telaye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**